assets/js/googlesitekit/data/create-notifications-store.js

```diff
@@ -63,11 +67,11 @@ export const createNotificationsStore = ( type, identifier, datapoint, {
 	const STORE_NAME = storeName || `${ type }/${ identifier }`;
 
 	const INITIAL_STATE = {
-		serverNotifications: undefined,
+		serverNotifications: server ? undefined : null,
```

Comment:
Why is this `null`? I think this can always be `undefined`, regardless of `server`. If server notifications are disabled, this is being ignored entirely I think? cc @tofumatt

Reply:
That's true, but I thought explicitly marking them as `null` was more "correct". We use `undefined`, generally, to indicate that something is loading/not-yet-available. By setting this to `null` explicitly, no code that checks for it being `undefined` would make any assumptions about the state of data or where it is in a notifications-request-flow.

I get that it doesn't _super_ matter here, because we don't load the resolver when `server` is false-y. But I thought it best to be as explicit as possible.

Reply:
(That said… an empty object would be fine too. Maybe even better.)

Reply:
Thanks @tofumatt, that makes sense. Let's go with an empty object since `null` is not a value this should ever have. Our `undefined` is for loading state, and `null` would work if this was actually an "object" - it's a map of objects and not a single object though (although of course technically an object), so the empty object is more appropriate here I think, since it's an empty map.

Reply:
Fixed